tools/replay/CMakeLists.txt

```diff
@@ -37,14 +37,30 @@ target_sources(gfxrecon-replay
                     $<$<BOOL:WIN32>:${CMAKE_SOURCE_DIR}/version.rc>
 )
 
-target_include_directories(gfxrecon-replay PUBLIC ${CMAKE_BINARY_DIR})
+if (WIN32)
+    target_include_directories(gfxrecon-replay PUBLIC ${CMAKE_BINARY_DIR} ../../external/json/win64/include)
+else()
+    target_include_directories(gfxrecon-replay PUBLIC ${CMAKE_BINARY_DIR} ../../external/json/linux/include)
+endif()
+
+if (WIN32)
+    target_link_directories(gfxrecon_decode
+                            PUBLIC
+                                ${CMAKE_SOURCE_DIR}/external/json/win64/lib)
```

Comment:
We usually include pre-compiled libraries and the header files in `external/precompiled/` directory.
I think headers + binaries should be moved there for consistency